socorro/processor/cache_manager.py

```diff
@@ -222,37 +222,39 @@ def _event_generator(self, nonblocking=False):
                 try:
                     for event in self.inotify.read(timeout=timeout):
                         processed_events = True
-                        event_flags = flags.from_mask(event.mask)
+                        event_mask = event.mask
 
-                        flags_list = ", ".join([str(flag) for flag in event_flags])
-                        self.logger.debug("EVENT: %s: %s", event, flags_list)
+                        # flags_list = ", ".join(
+                        #     [str(flag) for flag in flags.from_mask(event_mask)]
+                        # )
+                        # self.logger.debug("EVENT: %s: %s", event, flags_list)
```

Comment:
This is some extra work and the loop shouldn't be doing this work _every_ event if it's not going to be logged. This is only helpful for development. Ergo I commented it out.